api/serializers.py

```diff
@@ -41,6 +41,31 @@ class Meta:
         )
 
 
+class CountUserSerializer(serializers.Serializer):
+    total_spec_user = serializers.SerializerMethodField()
+    photo_user = serializers.SerializerMethodField()
+    video_user = serializers.SerializerMethodField()
+
+    class Meta:
+        fields = (
+            'total_spec_user',
+            'photo_user',
+            'video_user',
+        )
+
+    def get_total_spec_user(self, services, *args, **kwargs):
+        users = User.objects.filter(
+            is_photographer=True
+        ) | User.objects.filter(is_video_operator=True)

```

Comment:
Это точно соберет всех специалистов?
```suggestion
        users = User.objects.filter(
            models.Q(is_photographer=True) |
            models.Q(is_video_operator=True)
        )

```

Reply:
Да я проверял. 
почему есть сомнения ? 
Можно работать как с объектам Q так и не сними.